templates/react-ts/build.js

```diff
@@ -0,0 +1,83 @@
+//
+// This is an example build script for Socket Runtime
+// When you run 'ssc build', this script (node build.js) will be run
+//
+import fs from 'node:fs'
+import path from 'node:path'
+
+import esbuild from 'esbuild'
+
+const cp = async (a, b) =>
+  fs.promises.cp(path.resolve(a), path.join(b, path.basename(a)), {
+    recursive: true,
+    force: true
+  })
+
+async function main() {
+  const prod = process.argv.find((s) => s.includes('--prod'))
+
+  const params = {
+    entryPoints: ['src/index.tsx'],
+    format: 'esm',
+    bundle: true,
+    minify: !!prod,
+    sourcemap: !!prod,
```

Comment:
lets make it `true`

Reply:
should source maps be included in production?

Reply:
not necessary, but I'm not opposed to that personally (previous discussion is [here](https://github.com/socketsupply/create-socket-app/pull/27#discussion_r1133333925))

Reply:
https://github.com/socketsupply/create-socket-app/pull/27#discussion_r1134007370

Reply:
Changed to 
```
    sourcemap: true,
```